sonic_y_cable/y_cable.py

```diff
@@ -92,6 +92,12 @@
 
 MAX_NUM_LANES = 4
 
+# definitions of firmware upgrade codes
+# to be returned by the user when calling the
+# firmware upgrade routine
```

Comment:
Suggest simplifying to `# Valid return codes for upgrade_firmware() routine`

Reply:
fixed